homeassistant/components/netgear/router.py

```diff
@@ -112,11 +113,19 @@ def _setup(self) -> bool:
         self.serial_number = self._info["SerialNumber"]
         self.mode = self._info.get("DeviceMode", MODE_ROUTER)
 
+        loaded_entries = list(self.hass.config_entries.async_entries(DOMAIN))
```

Comment:
We can remove the copy to list.
```suggestion
        loaded_entries = self.hass.config_entries.async_entries(DOMAIN)
```

Reply:
We need to check the state of the config entry if we only want loaded entries.

Reply:
What if all other config entries are disabled?

Reply:
Well I guess we schould still track devices if the other Netgear integrations are disabled.
So we schould not count disabled or discovered but not configured devices.
However I don't see those status in the ConfigEntryState enum:
https://github.com/home-assistant/core/blob/dbd3ca5ecd8ee0119f6225751f9abf0efc2f0cb1/homeassistant/config_entries.py#L78-L92

Reply:
The `disabled_by` attribute on the config entry describes the disabled status.

https://github.com/home-assistant/core/blob/ab82f71b4315d792d80eb1318ae8899350a8d3c5/homeassistant/config_entries.py#L259

Reply:
I added a commit to check if disabled_by is None (so not disabled).